app/src/main/res/layout/activity_traffic_violations.xml

```diff
@@ -167,20 +167,6 @@
         app:srcCompat="@drawable/backarrow"
         tools:ignore="RedundantDescriptionCheck" />
 
-    <ImageButton
```

Comment:
Reimplements the screen without the refresh button as discussed as it was an issue we were facing